lib/main.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:flutter_dotenv/flutter_dotenv.dart';
-import 'package:nativemoon/routes.dart';
+import 'package:NativeMoon/routes.dart';
```

Comment:
why

Reply:
unnecessary indeed, not what i wanted to do. already being reverted.